src/groupeddataframe/grouping.jl

```diff
@@ -429,9 +461,13 @@ function combine(f::Any, gd::GroupedDataFrame)
         return gd.parent[1:0, gd.cols]
     end
 end
+
 combine(gd::GroupedDataFrame, f::Any) = combine(f, gd)
-combine(gd::GroupedDataFrame, f::Pair...) = combine(f, gd)
-combine(gd::GroupedDataFrame, f::Pair) = combine(f, gd)
+
+combine(gd::GroupedDataFrame, f::Union{Pair, AbstractVector{<:Pair}}...) =
+    combine(reduce(vcat, f), gd)
+
+combine(gd::GroupedDataFrame, f::NamedTuple) = combine(f, gd)
```

Comment:
Sorry if I've asked before, but why is that method needed?

Reply:
It's not needed. The user can always splat the keyword arguments.

Reply:
Update: the user does not need to splat the keyword arguments for this to work.  This is because the keyword arguments are splatted in the function call already.